### PR TITLE
fix: social media and focus in footer links

### PR DESCRIPTION
**Pull request by GustavoUT22**

### CODE REVIEW

**Comment by Mardecera** [comment]
Focus behavior is inappropriate, it paints an additional black line. And the design of the focus on the `network buttons` does not match that of `Figma`.

Captura de pantalla 2024-05-21 071839  Captura de pantalla 2024-05-21 071752

**Review by Mardecera — CHANGES_REQUESTED** [review]
Focus behavior is inappropriate, it paints an additional black line. And the design of the focus on the `network buttons` does not match that of `Figma`.

Captura de pantalla 2024-05-21 071839  Captura de pantalla 2024-05-21 071752

File: src/components/SocialMedia/SocialMedia.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const socialLinksIcons: SocialLinks[] = [
   {
     id: '6',
     to: 'twitter.com/binland_dev',
```
Comment (Mardecera) [inline src/components/SocialMedia/SocialMedia.tsx:40]: @BinlandDev

File: src/components/SocialMedia/SocialMedia.tsx [hunk 2]
```diff
@@ -5,29 +5,29 @@ import {
...
-  to: string
-  icon: React.ReactNode
-}
+import { SocialMediaWrapper } from './SocialMedia.styles'
+import { ILinks } from 'types/ui/Link'
 
 const ICON_SIZE = 24
 
-const socialLinksIcons: SocialLinks[] = [
+const socialLinksIcons: ILinks[] = [
```
Comment (Mardecera) [inline src/components/SocialMedia/SocialMedia.tsx:16]: Singular ILink

Comment (GustavoUT22) [reply src/components/SocialMedia/SocialMedia.tsx]: done

File: src/components/SocialMedia/SocialMedia.tsx (hunk 1 above)
Comment (GustavoUT22) [reply src/components/SocialMedia/SocialMedia.tsx]: done

**Review by Mardecera — APPROVED** [review]
Nice!

**Review by CrhistianT7 — APPROVED** [review]
LGTM!!

**Pull request merged**